lib/file-helper.js

```diff
@@ -87,7 +87,7 @@ exports.getFingerprint = function (file) {
   }
 
   // Look for a fingerprinted asset at this path relative to the site root
-  return '/' + fingerprints[file]
+  return '/' + slash(fingerprints[file])
```

Comment:
But always need forward slashes in attributes `href="{{ getFingerprint('example.css') }}"`